backend/app/migrator/mapper_test.go

```diff
@@ -0,0 +1,41 @@
+package migrator
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestUrlMapper(t *testing.T) {
```

Comment:
I'd expect this test to be a table test checking many differnt inputs. Probably it could check all posible use case, including bad inputs